tests/phpunit/Structure/ResourcesAccessibilityTest.php

```diff
@@ -25,6 +25,9 @@ class ResourcesAccessibilityTest extends \PHPUnit\Framework\TestCase {
 	 * @dataProvider moduleDataProvider
 	 */
 	public function testModulesScriptsFilesAreAccessible( $modules ) {
+		if ( version_compare( MW_VERSION, '1.41', '>=' ) ) {
+			$this->markTestSkipped( 'Skipped test for MW 1.41+, check ResourceLoader.' );
+		}
 		$resourceLoader = MediaWikiServices::getInstance()->getResourceLoader();
 		$context = ResourceLoaderContext::newDummyContext();
```

Comment:
This can be fixed by doing something like:

https://github.com/SemanticMediaWiki/SemanticResultFormats/commit/ace37f7fa101d22c74bc33f992d8fda9977ea01f#diff-a76cde2667d375f0f8c1ba641bb9bc9e5cf94c932401f24867fc0a3eb9ff12fd

and

https://github.com/SemanticMediaWiki/SemanticResultFormats/pull/896/files#diff-a76cde2667d375f0f8c1ba641bb9bc9e5cf94c932401f24867fc0a3eb9ff12fd

Reply:
@paladox the solutions from your comment above work well. Thanks!